tests/Fixtures/Model/Person.php

```diff
@@ -17,4 +19,20 @@ public function setFullName(string $fullName): void
     {
         $this->fullName = $fullName;
     }
+
+    /**
+     * @return int
+     */
```

Comment:
We should configure the CS-Fixer to remove this type of useless PhpDoc (in a followup PR).